python/ray/tune/examples/cifar10_pytorch.py

```diff
@@ -206,21 +222,14 @@ def main(num_samples=10, max_num_epochs=10, gpus_per_trial=2):
     print("Best trial final validation accuracy: {}".format(
         best_trial.last_result["accuracy"]))
 
-    best_trained_model = Net(best_trial.config["l1"], best_trial.config["l2"])
-    device = "cpu"
-    if torch.cuda.is_available():
-        device = "cuda:0"
-        if gpus_per_trial > 1:
-            best_trained_model = nn.DataParallel(best_trained_model)
-    best_trained_model.to(device)
-
-    checkpoint_path = os.path.join(best_trial.checkpoint.value, "checkpoint")
+    if args.server_address:
+        # If using Ray Client, we want to make sure checkpoint access
+        # happens on the server. So we wrap `test_best_model` in a Ray task.
+        ray.get(ray.remote(test_best_model).remote())
+    else:
+        test_best_model()
```

Comment:
hmm this example is user-facing (is used in docs). but this method is not shown -- do you need to revise the docs (or define the function within the __main_end__ scope?

Reply:
Ah good point, updated